ee/codegen_integration/fixtures/simple_inline_subworkflow_node/code/display/nodes/subworkflow_node/__init__.py

```diff
@@ -16,6 +18,9 @@ class SubworkflowNodeDisplay(BaseInlineSubworkflowNodeDisplay[SubworkflowNode]):
     port_displays = {
         SubworkflowNode.Ports.default: PortDisplayOverrides(id=UUID("fa5c22bc-2499-43fa-880f-75fb20d0587f"))
     }
+    output_display = {
+        SubworkflowNode.Outputs.final_output: NodeOutputDisplay(id=UUID("6ab3665f-881d-488b-9124-a6da40136c68"), name="final_output")
```

Comment:
The display classes need to know what the subworkflow node's output IDs are.
This may be unnecessary if we fix the output display issue mentioned in the card in this PR's description, since this ID should be pulled from the workflow's outputs